implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/Tuple.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2017 Pivotal Software, Inc.
```

Comment:
2019?

Reply:
the rest of the dynatrace code was added in 2018, still has 2017 in the license files though

I'll change this one